manifests/nodes/sac92-isaac-base.pp

```diff
@@ -129,14 +129,24 @@
   }
 
   #Database Backup
-  file { '/local/data/rutherford/database-backup':
-    ensure => 'directory',
-    owner  => 'postgres',
-    group  => 'isaac',
-    mode   => '0755',
+  if ( $::fqdn =~ /(\w+-)?isaac-live/ ) {
```

Comment:
We've discussed it, and until we organise partitions on dev and staging, we want to treat them differently.